### PR TITLE
cap the number of considered message channels in PacketQueue

### PR DESCRIPTION
**Pull request by birarda**
https://highfidelity.fogbugz.com/f/cases/20051/Current-message-cycling-can-be-problematic-is-new-messages-are-added-faster-than-the-SendQueue-can-clear-them

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16076/HighFidelity-Beta-PR14621-370713d.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16076/HighFidelity-Beta-PR14621-370713d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16076/HighFidelity-Beta-Interface-PR14621-370713d.dmg

**Comment by antoninahifi** [comment]
FAIL
Tested with clumsy filtering set to 500ms delay and 10% drop in distributed3 domain with 200 bots https://gist.githubusercontent.com/birarda/d91a2cefaaf408a094cc454072bb77cc/raw/17a42cb843dcaca40224b2a7d8168f7b5fcc3383/frequent-updates-bot.js
Result: White spheres issue still present

**Comment by howard-stearns** [comment]
(DNM for the moment, even if it passes, because we have some other testing to do in which we don't want this in master.)

**Comment by howard-stearns** [comment]
Under the lighter clumsy settings of 100ms delay and no drops (which nonetheless do reproduce white spheres in master), this PR _does_ eventually resolve the white spheres. That is, when you arrive in the domain, all bots are white spheres. But they start to load in after about 10 seconds, and are completely resolved after a minute or so (while retaining the packet delay).

**Comment by howard-stearns** [comment]
This is one of five fixes that are being evaluated against server-under-load issues:
https://github.com/highfidelity/hifi/pull/14601
https://github.com/highfidelity/hifi/pull/14621
https://github.com/highfidelity/hifi/pull/14633
https://github.com/highfidelity/hifi/pull/14627
https://github.com/highfidelity/hifi/pull/14558
Please Do Not Merge until either:
- all five have been evaluated for how they do under server load, or
- you need the fix right away and check with Brad

**Comment by howard-stearns** [comment]
server under load testing:
RESOLVES ALL WHITE SPHERES (with moderate 100ms delay and no drops)

**Comment by antoninahifi** [comment]
PASS

File: libraries/networking/src/udt/PacketQueue.h [hunk 1]
```diff
@@ -13,6 +13,7 @@
 #define hifi_PacketQueue_h
 
 #include <list>
+#include <queue>
```
Comment (Atlante45) [inline libraries/networking/src/udt/PacketQueue.h]: Unnecessary header

**Comment by howard-stearns** [comment]
Removing the DNM that was placed until the "five fixes" were concurrency-tested.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16337/HighFidelity-Beta-PR14621-3c6b7e3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16337/HighFidelity-Beta-Interface-PR14621-3c6b7e3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16337/HighFidelity-Beta-PR14621-3c6b7e3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16337/HighFidelity-Beta-Interface-PR14621-3c6b7e3.exe

**Pull request merged**